x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/EnrichPlugin.java

```diff
@@ -126,15 +126,19 @@
         return String.valueOf(maxConcurrentRequests * maxLookupsPerRequest);
     }, val -> Setting.parseInt(val, 1, Integer.MAX_VALUE, QUEUE_CAPACITY_SETTING_NAME), Setting.Property.NodeScope);
 
+    public static final Setting<Long> CACHE_SIZE = Setting.longSetting("enrich.cache_size", 1000, 0, Setting.Property.NodeScope);
```

Comment:
Were you planning to add documentation for this new setting?

Reply:
Good point, I will add docs for this.

Reply:
I pushed: 8c328a44977bfce9b0d4e79112aaec001b1521d0
This is the first enrich setting that we document.
@jrodewig do you think this is the right place?

Reply:
Maybe other enrich settings should be documented as well along side the cache size setting. I will do this in a follow up and ping James when he is back. I will revert the mentioned commit for now.

Reply:
Thanks for the ping @martijnvg. I'll be happy to take a look!